corehq/apps/cloudcare/utils.py

```diff
@@ -3,8 +3,7 @@
 
 
 def should_show_preview_app(request, domain_obj, username):
-    live_preview_ab = ab_tests.ABTest(ab_tests.LIVE_PREVIEW, request)
-    if domain_obj.is_onboarding_domain and live_preview_ab.version == ab_tests.LIVE_PREVIEW_ENABLED:
+    if domain_obj.is_onboarding_domain:
         return True
     elif domain_obj.is_onboarding_domain and not request.couch_user.is_dimagi:
```

Comment:
acutally i misread this. it wouldn't be as bad as i suggested, but still can just drop the whole thing